helm-chart/templates/api-deployment.yaml

```diff
@@ -85,6 +85,8 @@ spec:
             {{ if .Values.debug.storage.useLocal }}
             - name: USER_ID
               value: "1000"
+            - name: GROUP_ID
```

Comment:
I'm not sure we need GROUP_ID in the helm-charts. It's an on-premise thing no?

Reply:
It's behind the `.Values.debug.storage.useLocal` flag`, so it will not be used on prod deployments